main.go

```diff
@@ -191,6 +192,12 @@ func getMetalConfig(providerConfig string) (metal.Config, error) {
 		config.AnnotationBGPPass = annotationBGPPass
 	}
 
+	config.AnnotationNetworkIPv4Private = metal.DefaultAnnotationNetworkIPv4Private
+	annotationNetworkIPv4Private := os.Getenv(envVarAnnotationIPv4NetworkPrivate)
```

Comment:
perhaps the env var constant could match the config name: envVarAnnotationNetworkIPv4Private (to make future code searches easier)

Reply:
That's a good idea. Will do.

Reply:
Done.